sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -1177,6 +1177,8 @@ public partial class BlobBaseClient
         public virtual System.Uri Uri { get { throw null; } }
         public virtual Azure.Response AbortCopyFromUri(string copyId, Azure.Storage.Blobs.Models.BlobRequestConditions conditions = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response> AbortCopyFromUriAsync(string copyId, Azure.Storage.Blobs.Models.BlobRequestConditions conditions = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.Storage.Blobs.Models.BlobCopyInfo> CopyFromUri(System.Uri source, Azure.Storage.Blobs.Models.BlobCopyFromUriOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Storage.Blobs.Models.BlobCopyInfo>> CopyFromUriAsync(System.Uri source, Azure.Storage.Blobs.Models.BlobCopyFromUriOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
All languages support this, it's a [backend feature](https://docs.microsoft.com/en-us/rest/api/storageservices/copy-blob-from-url).

How about ```CopyFromUriSync()```?

Reply:
`CopyFromUriSync` wouldn't work well because then we'd have `CopyFromUriSync` and `CopyFromUriSyncAsync` overloads.  I think it would have to be more like `SyncCopyFromUri`/`SyncCopyFromUriAsync` or `ImmediateCopyFromUri`/`ImmediateCopyFromUriAsync`.

I don't see this property in the Java API, but I don't know if I'm looking at their latest.  Same for Python.  It looks like JS calls this `syncCopyFromUrl`

Reply:
I think they may have it as a parameter on the StartCopyFromUri API.

Reply:
I think the approach you have here will be easier for users because it makes the calling convention more obvious.  I'd just vote for a name like `SyncCopyFromUri` to match what others have done and avoid confusion with all the other `CopyFromUri` methods.

Reply:
Will do, thanks for the feedback.